### ISSUE TITLE
Atom freezes if <html lang=""> is left empty

### ISSUE THREAD

**Issue by benoz23** [emmetio/emmet-atom#428]
This is a very particular (and minor) issue, but it made me waste a lot of time to figure out what was going on.

I work on a mac running OS X El Capitan 10.11.4, with Atom 1.7.3 installed and Emmet-atom 2.4.3.

If `lang` attribute in `<html>` tag is left empty **AND** you try to add a new line at the very beginning of the document by pressing enter right before `<!DOCTYPE html>` declaration, Atom will freeze, leading to "Editor is not responding" alert.

Disabling Emmet prevented the problem from recurring. I tried to uninstall Atom (with [AppCleaner](https://freemacsoft.net/appcleaner/)) and remove all packages (deleting .atom folder). After i made a fresh new install of Atom, the issue was gone. But as soon as i installed Emmet (as first and only package), Atom started to freeze again.

Filling `lang` attribute fixed the problem.

**Comment by jcuenod** [comment]
Ja, pretty sure this is the same issue - I narrowed it down to trying to add a new line in the php tags in this:

```
<?php

?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

```

**Comment by burakerdem** [comment]
It's same here with Atom v1.8.0. When Emmet v2.4.3 is disabled, everything works fine.

**Comment by sergeche** [comment]
OK, will check it out
